### DISCUSSION TITLE
Aqara RGB Effects for the RGB Lights

### DISCUSSION THREAD

**Discussion by LZXCorp** [Koenkk/zigbee2mqtt#22215] [Devices]
From [CL-L02D](https://www.zigbee2mqtt.io/devices/CL-L02D.html), it does not include any option to change effects.

> light (state, brightness, color_temp), power_outage_count, device_temperature, power_outage_memory, light (state, brightness, color_temp, color_xy, color_hs), linkquality

Would it be possible to add an option to use the preset RGB effects?

As for the [LGYCDD01LM](https://www.zigbee2mqtt.io/devices/LGYCDD01LM.html), would it be possible to further expand the functionality of the color effects such that I can create my own presets?

**Comment by jjnj023** [comment]
Im curious too. I can controll the color of the Aqara T1M RGB ring, but not any animation. I cannot find it in Z2M or in home assistant

**Comment by johnsturgeon** [comment]
Can this be a formal issue?  Or is the 'discussion' enough here?

Welp, I bit the bullet and added an issue / feature request
https://github.com/Koenkk/zigbee2mqtt/issues/22247

**Reply by LZXCorp** [reply]
For the case of the [CL-L02D](https://www.zigbee2mqtt.io/devices/CL-L02D.html), then yeah a feature request yes.

As for the LGYCDD01LM,
> As for the [LGYCDD01LM](https://www.zigbee2mqtt.io/devices/LGYCDD01LM.html), would it be possible to further expand the functionality of the color effects such that I can create my own presets?

The LGYCDD01M has been release since 6+ months ago and we've gotten up from just changing colours to having preset effects. However, I am more interested towards whether creating custom effects straight from Home Assistant or Z2M is possible.
